### PR TITLE
Hotfix/keys menu improvements

### PR DESCRIPTION
**Pull request by hjoaquim**
# Description

Goals:
1. Avoid re-checking the keys' status every time
2. Introduce `status` cmd that can force one or all keys status to reevaluate
3. Improve the `mykeys`'s `show` flag help text.

### CODE REVIEW

File: openbb_terminal/keys_controller.py [hunk 1]
```diff
@@ -137,6 +146,44 @@ def call_mykeys(self, other_args: List[str]):
...
+
+        ns_parser = self.parse_known_args_and_warn(
+            parser, other_args, export_allowed=EXPORT_ONLY_RAW_DATA_ALLOWED
+        )
+
+        is_key = bool(ns_parser and ns_parser.key)
+
+        if is_key:
+            if ns_parser.key in self.API_LIST:
+                self.status_dict[ns_parser.key] = self.get_check_keys_function(
```
Comment (hjoaquim) [inline openbb_terminal/keys_controller.py:178]: @montezdesousa should this be in a try except?

Comment (montezdesousa) [reply openbb_terminal/keys_controller.py:178]: Yes, some check functions don't handle that properly

File: openbb_terminal/keys_controller.py [hunk 2]
```diff
@@ -137,6 +146,53 @@ def call_mykeys(self, other_args: List[str]):
...
+        parser.add_argument(
+            "-k",
+            "--key",
+            type=str,
+            dest="key",
+            help="Force the status to reevaluate for a specific key.",
+            default=None,
+        )
+        if other_args and "-f" in other_args[0]:
+            other_args.insert(1, "True")
```
[the remark this answers is not in the document]
Comment (hjoaquim) [reply openbb_terminal/keys_controller.py]: does not apply anymore. the --force arg was removed

File: openbb_terminal/keys_controller.py [hunk 3]
```diff
@@ -137,6 +146,53 @@ def call_mykeys(self, other_args: List[str]):
...
+
+        ns_parser = self.parse_known_args_and_warn(
+            parser, other_args, export_allowed=EXPORT_ONLY_RAW_DATA_ALLOWED
+        )
+
+        if ns_parser:
+            if ns_parser.key:
+                if ns_parser.key in self.API_LIST:
```
[the remark this answers is not in the document]
Comment (hjoaquim) [reply openbb_terminal/keys_controller.py]: did it here: https://github.com/OpenBB-finance/OpenBBTerminal/pull/4699/commits/a45dd6389880b7ef7909b341729290f7ecfe813e

**Review by montezdesousa — APPROVED** [review]

**Pull request merged**